templates/sidebar.html

```diff
@@ -130,15 +131,21 @@
                         <li class="nav-item">
                             <h5>Name: <span class="lead"> {{ profile.get_full_name }}</span> </h5>
                         </li>
+                    {% endif %}
+                    
+                    {% if volun.desig is None %}
                         <li class="nav-item">
-                            <h5>Roll No: <span class="lead">{{ volun.roll_no }}</span> </h5>
+                            <h5>Designation: <span class="lead">Volunteer</span> </h5>
```

Comment:
Remove the titles (like Name, Designation) and keep the values only. Also, you can remove the span element and put the values in h5 element only (removing the margins of h5). Also, having the text color of designation as red would look good, I think.